ci/infra/azure/variables.tf

```diff
@@ -0,0 +1,175 @@
+variable "azure_location" {
+  type = string
+  # default     = "West Europe"
+  description = "Name of the AZURE location to be used"
+}
+
+variable "enable_zone" {
+  type        = bool
+  default     = false
+  description = "Use this if only zone is available in the deploying region"
+}
+
+variable "azure_availability_zones" {
+  type        = list(string)
+  default     = ["1", "2", "3"]
+  description = "List of Availability Zones (e.g. `['1', '2', '3']`)"
+}
+
+variable "cidr_block" {
+  type        = string
+  default     = "10.1.0.0/16"
+  description = "CIRD blocks for virtual_network"
+}
+
+variable "bastionhost_subnet_cidr" {
+  type        = string
+  default     = "10.1.1.0/24"
+  description = "CIDR blocks for the bastion host subnet"
+}
+
+variable "private_subnet_cidr" {
+  type        = string
+  default     = "10.1.4.0/24"
+  description = "Private subnet of virtual_network"
+}
+
+variable "stack_name" {
+  type        = string
+  default     = "k8s"
+  description = "Identifier to make all your resources unique and avoid clashes with other users of this terraform project"
+}
+
+variable "dnsdomain" {
+  type        = string
+  default     = ""
+  description = "Name of DNS domain"
+}
+
+variable "authorized_keys" {
+  type        = list(string)
+  default     = []
+  description = "SSH keys to inject into all the nodes"
+}
+
+variable "ntp_servers" {
+  type        = list(string)
+  default     = []
+  description = "List of NTP servers to configure"
+}
+
+variable "dns_nameservers" {
+  type        = list(string)
+  default     = []
+  description = "List of Name servers to configure"
+}
+
+variable "repositories" {
+  type        = map(string)
+  default     = {}
+  description = "Maps of repositories with '<name>'='<url>' to add via cloud-init"
+}
+
+variable "packages" {
+  type        = list(string)
+  default     = []
+  description = "List of packages to install"
+}
+
+variable "username" {
+  type        = string
+  default     = "sles"
+  description = "Username for the cluster nodes"
+}
+
+variable "password" {
+  type        = string
+  default     = "linux"
+  description = "Password for the cluster nodes"
+}
+
+variable "caasp_registry_code" {
+  type        = string
+  default     = ""
+  description = "SUSE CaaSP Product Registration Code"
+}
+
+variable "rmt_server_name" {
+  type        = string
+  default     = ""
+  description = "SUSE Repository Mirroring Server Name"
+}
+
+variable "suma_server_name" {
+  type        = string
+  default     = ""
+  description = "SUSE Manager Server Name"
+}
+
+variable "create_bastionhost" {
+  type        = bool
+  description = "Enables creation of bastion host"
+  default     = false
+}
+
+variable "masters" {
+  default     = 1
```

Comment:
Let's make it `3` which is our recommended size for production deployments.

Reply:
changed